agent/util-scripts/pbench-tool-meister-start

```diff
@@ -1151,12 +1156,14 @@ if __name__ == "__main__":
         default=os.environ.get("PBENCH_REDIS_SERVER", None),
         help=(
             "Specifies the IP/port to use for the Redis server - if not"
-            " present, the defaults are used; can be present in two forms:"
+            " present, the defaults are used,"
+            f" ${{_pbench_full_hostname}}:{def_redis_port};"
+            " the specified value can take either of two forms:"
```

Comment:
Nice addition, but the syntax is cumbersome.  I would have suggested isolating the f-string to just the bit which needs formatting:
```
            "Specifies the IP/port to use for the Redis server - if not"
            " present, the defaults are used, ${_pbench_full_hostname}:"
            f"{def_redis_port};"
            " the specified value can take either of two forms:"
```